### PR TITLE
Ultimate PC Speakers

### PR DESCRIPTION
**Pull request by Meesmeesie**
## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (
Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

[meesmeesie.md](https://github.com/hackclub/winter/files/10259674/meesmeesie.md)

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
hey @Meesmeesie thanks for submitting a PR and sharing your project! could you please add back the exu3.md file you deleted?

also, to confirm, are you following this guide? https://diyperks.com/the-ultimate-pc-sound-system/ if so, can you please expand on it within your plan as to what you'll actually be doing?

and, do you have all the equipment you need to build this? like machinery to cut the wood and soldering tools?

could you please also factor in shipping and taxes? 

thank you so much!

**Comment by bellesea** [comment]
> hey @Meesmeesie thanks for submitting a PR and sharing your project! could you please add back the exu3.md file you deleted?
> 
> also, to confirm, are you following this guide? https://diyperks.com/the-ultimate-pc-sound-system/ if so, can you please expand on it within your plan as to what you'll actually be doing?
> 
> and, do you have all the equipment you need to build this? like machinery to cut the wood and soldering tools?
> 
> could you please also factor in shipping and taxes?
> 
> thank you so much!

hiii @Meesmeesie hope you're doing awesome! checking in to ask: are you still interested in moving forward on this project?

**Comment by cfanoulis** [comment]
Hi @Meesmeesie! Thanks for your proposal. It's been a week since Belle's message and we want to know if you're still interested in the grant. Please reply in the next 48 hours to this PR (or this email) if you're still interested, else we'll be closing this proposal. Hacky New Year!

**Comment by Meesmeesie** [comment]
Hi, I was busy with school. But I'm still interested so I'm going to change
this now!

Op za 7 jan. 2023 om 20:43 schreef Charalampos Fanoulis <
***@***.***>:

**Comment by Meesmeesie** [comment]
i'm going to follow the guide on that website and the full build guide
video on youtube (https://youtu.be/408HxcN2gw4), and I (my dad :)) have all
the tools we need. My dad and I are going to make this together, because it
is hard to do it alone!
And i have a question, I live in the Netherlands, and really don't know the
shipping and taxes! Can you look that up for me? And if it becomes too
pricey, I can buy the wood myself!

**Comment by bellesea** [comment]
@Meesmeesie great! could you please add that you'll be referencing both links within your file then we should be good to move forward?

also, for shipping and taxes that's calculated on checkout of the individual items you listed in your budget so if you click on the link to purchase and move forward to checkout then insert your address you should be able to see the shipping and taxes. this just helps ensure we give you enough hehe :) if you need more help with this, don't worry about it and we can work with you on this later on.

thank you!

**Comment by Meesmeesie** [comment]
I’m done!

File: meesmeesie.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+name: "@Meesmeesie"
+project: "Ultimate PC Sound System"
+---
+
+# Hack Club Bank Sign
```
Comment (bellesea) [inline meesmeesie.md:6]:
```suggestion
# Ultimate PC Sound System
```

**Comment by bellesea** [comment]
@Meesmeesie thanks for pushing that! i'm looking over the build guide and noticed that two transformers and speakers are needed but noticed you only had one of each in your budget? did you plan on only doing one and modifying the design? also, do you have the wires you need?

the person also got a 4 channel amp as well as a digital to analog converter (or an auxiliary port connected directly to amps), do you need these?

and, for building your audio filter do you have the capacitors etc.?

thank you!

**Comment by bellesea** [comment]
hey @Meesmeesie following up on my last message, could you please respond to it within the next 48 hours if you're interested in proceeding? thank you sm :)

**Comment by bellesea** [comment]
hey @Meesmeesie i'll be closing this PR for now as it's been two days since my message. Since the deadline has passed, please re-open this PR within 48 hours if you're interested in proceeding or we won't be able to review your PR anymore. thank you so much!

**Pull request closed without merging**